configs/platforms/osx-10.12-x86_64.rb

```diff
@@ -0,0 +1,14 @@
+platform "osx-10.12-x86_64" do |plat|
+  plat.servicetype 'launchd'
+  plat.servicedir '/Library/LaunchDaemons'
+  plat.codename "sierra"
+
+  plat.provision_with 'curl http://pl-build-tools.delivery.puppetlabs.net/osx/homebrew_sierra.tar.gz | tar -x --strip 1 -C /usr/local -f -'
```

Comment:
just a heads up that the homebrew_sierra tarball was just a quick way to get us building.  we were using a much older homebrew tarball that did not have definitions for sierra.  this more recent tarball is a lot more verbose ( Homebrew/brew@27b0ca4 ) about running builds as root ( as in don't do it ).